rasa/utils/common.py

```diff
@@ -195,6 +195,14 @@ def sort_list_of_dicts_by_first_key(dicts: List[Dict]) -> List[Dict]:
     return sorted(dicts, key=lambda d: list(d.keys())[0])
 
 
+def transform_collection_to_sentence(collection: Collection[Text]) -> Text:
+    """Transforms e.g. a list like ['A', 'B', 'C'] into a sentence 'A, B and C'."""
+    x = [item for item in collection]
```

Comment:
that's just `lis
```suggestion
    x = list(collection)
```

Reply:
do we actually need this part? map should work on every collection no?

Reply:
but `x[:-1]` doesn't work for e.g. sets

Reply:
mhm :+1: then how about `list(collection)`?

Reply:
yep, that's how I changed it now